src/components/Header/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
+import classes from './header.module.css';
```

Comment:
Can you rename this to styles for consistency? `import styles from './header.module.css'`